command-snapshot.json

```diff
@@ -1,340 +1,1037 @@
 [
-  {
```

Comment:
View this w/ whitespace diffs off to see the real changes. The commands should support all the same flags as before. `--targetusername` and `--apiversion` don't show in this file, but are supported via the flags configs the commands are using.